lib/archethic/metrics/helpers.ex

```diff
@@ -145,6 +183,24 @@ defmodule ArchEthic.Metrics.Helpers do
     end
   end
 
+  @doc """
+  Recieves [[node1_metrics],[node2_metrics],[node3_metrics],...]
+  Recieves List of list of metrics , each metric is a map with name and type.
+  [[node1_metrics] , [node2_metrics] , ... ] =>
+    [
+       [ %{metrics: [],name: "metric_name", type: "metric_name"}] ,
+       [ %{metrics: [],name: "metric_name", type: "metric_name"}] ,
+       ...
+   ]
+   Returns [[node1_metrics],[node2_metrics],[node3_metrics],...]
+   Returns list of list of metrics , where each metric is a map
+   with name and count and sum for histogram type metrics , for
+   metric type guage it is a map with %{"metric_name" => 0}.
+   [
+     [%{"metric_name" => %{ count: 0, sum: 0 }} , %{"metric_name" => %{count: 0, sum: }}], ...],
+     [%{"metric_name" => %{ count: 0, sum: 0 }} , %{"metric_name" => %{count: 0, sum: }}], ...],
+     ...]
+  """
```

Comment:
I suggest you to leverage doctests, so your code will match your actuals and expectations